test/extensions/dynamic_modules/http/integration_test.cc

```diff
@@ -64,7 +115,13 @@ TEST_P(DynamicModulesIntegrationTest, PassThrough) {
   EXPECT_EQ(10U, response->body().size());
 }
 
-TEST_P(DynamicModulesIntegrationTest, HeaderCallbacks) {
+TEST_P(DynamicModulesIntegrationTest, HeaderCallbacks) { runHeaderCallbacksTest(false); }
+
+TEST_P(DynamicModulesIntegrationTest, HeaderCallbacksWithUpstreamFilter) {
+  runHeaderCallbacksTest(true);
+}
+
+TEST_P(DynamicModulesIntegrationTest, UpstreamFilter) {
```

Comment:
how does this test differ from the above `
TEST_P(DynamicModulesIntegrationTest, HeaderCallbacks) { runHeaderCallbacksTest(false); }` ? would it be possible to delete this one?

Reply:
yeah, forgot to cleanup. Removed. Thanks!